src/Plugin/Reverse/Reverse.cpp

```diff
@@ -0,0 +1,237 @@
+/*
+  ZynAddSubFX - a software synthesizer
+
+  Reverse.cpp - DPF + Zyn Plugin for Reverse
+  Copyright (C) 2015 Filipe Coelho
+  Author: Filipe Coelho
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+*/
+
+// DPF includes
+#include "../AbstractFX.hpp"
+
+// ZynAddSubFX includes
+#include "Effects/Reverse.h"
+
+/* ------------------------------------------------------------------------------------------------------------
+ * Reverse plugin class */
+
+class ReversePlugin : public AbstractPluginFX<zyn::Reverse>
+{
+public:
+    ReversePlugin()
+        : AbstractPluginFX(9, 5) {}
+
+    void setSpeedfactor(float factor)
+    {
+        effect->speedfactor=factor;
+    }
+
+protected:
+   /* --------------------------------------------------------------------------------------------------------
+    * Information */
+
+   /**
+      Get the plugin label.
+      This label is a short restricted name consisting of only _, a-z, A-Z and 0-9 characters.
+    */
+    const char* getLabel() const noexcept override
+    {
+        return "Reverse";
+    }
+
+   /**
+      Get an extensive comment/description about the plugin.
+    */
+    const char* getDescription() const noexcept override
+    {
+        // TODO
+        return "";
```

Comment:
Maybe fill that out and remove the TODO?

Reply:
ok